app/controllers/admin/users_controller.rb

```diff
@@ -49,6 +49,16 @@ def user_status
       redirect_to "/admin/users/#{@user.id}/edit"
     end
 
+    def export_data
+      user = User.find(params[:id])
+      using StringToBool
+      email_to_send = params[:send_to_admin].to_bool ? SiteConfig.email_addresses[:default] : user.email
+      ExportContentWorker.perform_async(user.id, email_to_send)
+      flash[:success] =
+        "Data exported to the#{params[:send_to_admin].to_bool ? 'admin' : 'user'}. The job will complete momentarily."
```

Comment:
as you already check `params[:send_to_admin].to_bool` up on line 55, maybe you can refactor to:

```ruby
send_to_admin = params[:send_to_admin].to_bool
if send_to_admin
  email = SiteConfig.email_addresses[:default]
  receiver = "admin"
else
  email = user.email
  receiver = "user
end
```

ps. there's a missing space between `the` and the category of user